GooglePlayInstant/Editor/PlayInstantQuickDeployInputManager.cs

```diff
@@ -23,97 +21,125 @@ namespace GooglePlayInstant.Editor
     /// </summary>
     public static class DeployBundleTextFields
     {
-        private const string dataStorageFilePath = ".google_play_instant_quick_deploy_inputs.json";
-
+        private const string DataStoragePrefix = "GooglePlayInstant.Editor.QuickDeploy.";
         private static string _oauth2CredentialsPath;
         private static string _localAssetBundlePath;
         private static string _remoteBucketName;
         private static string _remoteObjectName;
 
         public static string Oauth2CredentialsPath
         {
-            get { return getUpdatedValue(ref _oauth2CredentialsPath); }
-            set { updateNewValue(ref _oauth2CredentialsPath, value); }
+            get
+            {
+                if (_oauth2CredentialsPath != null)
+                {
+                    return _oauth2CredentialsPath;
+                }
+
+                UpdateFieldFromPrefs(ref _oauth2CredentialsPath, "oauth2CredentialsPath");
```

Comment:
"oauth2CredentialsPath" should be a const field and used here and below.

Reply:
Resolved. Abandoned this code since the config pull request addresses this